src/main/java/uk/gov/di/ipv/service/ConfigurationService.java

```diff
@@ -29,9 +29,20 @@ public String getAuthCodesTableName() {
         return System.getenv("AUTH_CODES_TABLE_NAME");
     }
 
-    public CredentialIssuers getCredentialIssuers() {
-        return CredentialIssuerLoader.loadCredentialIssuers(
-                ssmProvider.get(System.getenv("CREDENTIAL_ISSUER_CONFIG_PARAMETER_STORE_KEY")));
+    public CredentialIssuers getCredentialIssuers(CredentialIssuers credentialIssuers) {
+
+        String credentialIssuerConfigBase64 =
+                ssmProvider.get(System.getenv("CREDENTIAL_ISSUER_CONFIG_PARAMETER_STORE_KEY"));
+
+        if (credentialIssuers == null
+                || !credentialIssuerConfigBase64.equals(credentialIssuers.getSource())) {
+            var credentialIssuersResult =
+                    CredentialIssuerLoader.loadCredentialIssuers(credentialIssuerConfigBase64);
+            credentialIssuersResult.setSource(credentialIssuerConfigBase64);
```

Comment:
I think if we do the `setSource` in the `CredentialIssuerLoader` after we could just do `return credentialIssuerLoader.loadCredentialIssuers(credentialIssuerConfigBase64)` here which might be a bit cleaner.

Reply:
done